docs/examples.rst

```diff
@@ -105,54 +99,47 @@ get_asset
-       "transaction": {
-           "asset": {
-               "data": null,
-               "divisible": false,
-               "id": "cab78dc6-1cb2-4bc0-8ec2-267dedb5fa0f",
-               "refillable": false,
-               "updatable": false
-           },
-           "conditions": [
-               {
-                   "amount": 1,
-                   "cid": 0,
-                   "condition": {
-                       "details": {
-                           "bitmask": 32,
-                           "public_key": "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15",
-                           "signature": null,
-                           "type": "fulfillment",
-                           "type_id": 4
-                       },
-                       "uri": "cc:4:20:zL3F_XLRs_snrfmdqSFPqEcu-bu1xF6636oSYpNWvIw:96"
-                   },
-                   "owners_after": [
-                       "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15"
-                   ]
-               }
-           ],
-           "fulfillments": [
-               {
-                   "fid": 0,
-                   "fulfillment": {
+       "asset": {
+           "data": null,
+           "id": "b87bcc5e5700807ec64b949e7e6f8bccc269d2c6bc3b302632b366e01bc13507"
+       },
+       "id": "b87bcc5e5700807ec64b949e7e6f8bccc269d2c6bc3b302632b366e01bc13507",
+       "inputs": [
+           {
+               "fulfillment": {
+                   "bitmask": 32,
+                   "public_key": "35qDXhZTUvna23NLc1hMfmrgPniBwPgNjko1VfQuD3vF",
+                   "signature": null,
+                   "type": "fulfillment",
+                   "type_id": 4
+               },
+               "fulfills": null,
+               "owners_before": [
+                   "35qDXhZTUvna23NLc1hMfmrgPniBwPgNjko1VfQuD3vF"
+               ]
+           }
+       ],
+       "metadata": null,
+       "operation": "CREATE",
+       "outputs": [
+           {
+               "amount": 1,
+               "condition": {
+                   "details": {
                        "bitmask": 32,
-                       "public_key": "35qDXhZTUvna23NLc1hMfmrgPniBwPgNjko1VfQuD3vF",
+                       "public_key": "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15",
                        "signature": null,
                        "type": "fulfillment",
                        "type_id": 4
                    },
-                   "input": null,
-                   "owners_before": [
-                       "35qDXhZTUvna23NLc1hMfmrgPniBwPgNjko1VfQuD3vF"
-                   ]
-               }
-           ],
-           "metadata": null,
-           "operation": "CREATE"
-       },
-       "version": 1
+                   "uri": "cc:4:20:zL3F_XLRs_snrfmdqSFPqEcu-bu1xF6636oSYpNWvIw:96"
+               },
+               "public_keys": [
+                   "EnE1QD5kBY9Zrsp2Ejsp7W7ZMFAcH75SqR9wz6WrUR15"
+               ]
+           }
+       ],
+       "version": "0.9.0.dev"
```

Comment:
Should we just rename this to `0.9.0` so that we have the CLI ready for the release and don't have to touch it anymore?